scripts/lib.sh

```diff
@@ -251,22 +273,63 @@ generate_dockerfile()
 
 	curlOptions=("-OL")
 	[ -n "${http_proxy:-}" ] && curlOptions+=("-x ${http_proxy:-}")
+
+	readonly dockerfile_template="Dockerfile.in"
 	readonly install_go="
 RUN cd /tmp ; curl ${curlOptions[@]} https://storage.googleapis.com/golang/go${GO_VERSION}.linux-${goarch}.tar.gz
 RUN tar -C /usr/ -xzf /tmp/go${GO_VERSION}.linux-${goarch}.tar.gz
 ENV GOROOT=/usr/go
 ENV PATH=\$PATH:\$GOROOT/bin:\$GOPATH/bin
 "
 
-	readonly dockerfile_template="Dockerfile.in"
+	# Rust agent
+	# rust installer should set path apropiately, just in case
+	local cmake_file="cmake-${CMAKE_VERSION}.tar.gz"
+	local cmake_dir="cmake-${CMAKE_VERSION}"
+	readonly install_cmake="
+RUN pushd /root; wget https://github.com/Kitware/CMake/releases/download/v${CMAKE_VERSION}/${cmake_file}; tar -zxf ${cmake_file}; cd ${cmake_dir}; ./bootstrap; make; make install; popd
```

Comment:
- Please change all `wget` calls to `curl` (Kata uses `curl` everywhere).
- Why can't we install the packaged version of cmake with `apt`/`dnf`?
- Please add backslashes after each semi-colon so that the lines are shorter, and thus easier to read and diff (here and elsewhere).

Reply:
cmake/musl version is too old on some distros, we have to install them from source code